packages/angular-cli/upgrade/version.ts

```diff
@@ -0,0 +1,114 @@
+import {CliConfig} from '../models/config';
+import {readFileSync, existsSync} from 'fs';
+import {stripIndents} from 'common-tags';
+import {bold, red, yellow} from 'chalk';
+import * as path from 'path';
+
+
+function _findUp(name: string, from: string) {
+  let currentDir = from;
+  while (currentDir && currentDir !== path.parse(currentDir).root) {
+    const p = path.join(currentDir, name);
+    if (existsSync(p)) {
+      return p;
+    }
+
+    currentDir = path.dirname(currentDir);
+  }
+
+  return null;
+}
+
+
+function _hasOldCliBuildFile() {
```

Comment:
It doesn't access any class property/method, so it doesn't need to be a version.